lib/modules/dosomething/dosomething_user/dosomething_user.module

```diff
@@ -103,9 +103,34 @@ function dosomething_user_menu() {
     'file' => 'dosomething_user.admin.inc',
     'weight' => 500,
   );
+
+    return $items;
+}
+
+function dosomething_user_menu_alter(&$items) {
+    $items['user/%'] = array(
+    'page callback' => 'dosomething_user_get_view',
+    'page arguments' => array(1),
+    'type' => MENU_CALLBACK,
+    'access callback' => TRUE,
+  );
+
   return $items;
 }
 
+function dosomething_user_get_view($id) {
+  if(is_numeric($id)) {
```

Comment:
its been a while since i've been writing code in phoenix, but shouldn't these `if` statements have a space?
 `if (is_numeric($id)) {`